File: API/Features/Reservations/Reservations/Dtos/PassengerWriteDto.cs
using System;
using API.Infrastructure.Interfaces;

namespace API.Features.Reservations.Reservations {

    public class PassengerWriteDto : IBaseEntity {

        // PK
        public int Id { get; set; }
        // FKs
        public Guid? ReservationId { get; set; }
        public int GenderId { get; set; }
        public int NationalityId { get; set; }
        // Fields
        public string Lastname { get; set; }
        public string Firstname { get; set; }
        public string Birthdate { get; set; }
        public string PassportNo { get; set; }
        public string PassportExpiryDate { get; set; }
        public string SpecialCare { get; set; }
        public string Remarks { get; set; }
        public bool IsBoarded { get; set; }

    }

}

File: FrontEnd/src/app/features/reservations/reservations/classes/dtos/form/passenger-write-dto.ts
import { Guid } from 'guid-typescript'

export interface PassengerWriteDto {

    reservationId: Guid
    genderId: number
    nationalityId: number
    occupantId: number
    lastname: string
    firstname: string
    birthdate: string
    passportNo: string
    passportExpiryDate: string
    remarks: string
    specialCare: string
    isBoarded: boolean

}
